roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -342,6 +342,16 @@
     ports:
     - port: 9300
 
+- name: Modify logging-{{ es_component }}-cluster service to be headless
+  oc_edit:
+    kind: service
+    name: "logging-{{ es_component }}-cluster"
+    namespace: "{{ openshift_logging_elasticsearch_namespace }}"
+    content:
+      spec.publishNotReadyAddresses: "{{ true | bool }}"
```

Comment:
`True` ?

Reply:
The 'replace' kept failing.  I'll try 'True' to see if it makes a diff.

Reply:
Only this chnage worked for this line, but annotations in the previous task works only with 'true'

Reply:
Would it make a difference if its not quoted?

Reply:
I tried that too did not seem to result in success